webui/react/src/utils/flatRun.ts

```diff
@@ -0,0 +1,71 @@
+import { deletableRunStates, killableRunStates, terminalRunStates } from 'constants/states';
+import { PermissionsHook } from 'hooks/usePermissions';
+import { FlatRun, FlatRunAction } from 'types';
+
+type FlatRunChecker = (flatRun: Readonly<FlatRun>) => boolean;
+
+type FlatRunPermissionSet = Pick<
+  PermissionsHook,
+  'canCreateExperiment' | 'canDeleteFlatRun' | 'canModifyExperiment' | 'canMoveFlatRun'
```

Comment:
question: is there a reason we have `canDeleteFlatRun` and `canMoveFlatRun` but not `canCreateExperiment` and `canModifyExperiment`? seems like this could be consistent.

Reply:
i understand the concern. The reason is that permission for runs and experiments are the same for `canCreateExperiment` and `canModifyExperiment`. I don't think it makes sense to make the same functions for runs.

Reply:
Personally I think it's more clear to have them use the same name, even if `canCreateFlatRun` and `canModifyFlatRun` just wraps `canCreateExperiment` and `canModifyExperiment` without any changes -- but not something I would block merge on

Reply:
ok i wrapped these for flatRuns as alias.